Resources/Private/Language/locallang_db.xlf

```diff
@@ -12,6 +12,14 @@
         <source>Login/logout form for website users</source>
       </trans-unit>
 
+      <trans-unit id="plugin.password">
+        <source>Edit password form</source>
```

Comment:
Please change to `Password edit form`, otherwise it sounds like people could edit their password form here. ;-)